Portfolio/index.html

```diff
@@ -8,6 +8,59 @@
 
   <body style="margin: 0; padding: 0">
     <!-- Write your code below for PORTFOLIO -->
+   <h1>Chauhan.</h1>
+    <ul>
+      <li>ABOUT</li>
+      <li>SKILLS</li>
+      <li>PROJECTS</li>
+      <LI>REVIEWS</LI>
+    </ul><br>
+  <img src="/phone.icon.png" alt="Dialer Logo"/><br>
+  <button>CONTACT ME</button><br>
+
+<section>
+   <h1>Abhishek Chauhan<h1><br>
+        <img src="/johndoe.png" alt="Profile Picture"/><br>
+    <h1>Front End<br> Developer</h1>
+    <h3>GET TO KNOW ME </h3>
+        <p>My name is Abhishek, and I'm a passionate front-end developer with 2 Years of experience crafting responsive and user-friendly websites and applications. 
+        I specialize in [specific technologies or frameworks, e.g., React, Vue, Angular], and I have a strong grasp of the modern web development ecosystem including HTML, CSS, JavaScript, and related tools.</p><br>
+    <ul>
+          <li>Name:Abhishek Chauhan</li>
```

Comment:
The opening tag for the 'section' element on line 22 is missing the closing angle bracket.